src/storage/src/redis_pkhashes.cc

```
@@ -0,0 +1,1279 @@
+  UpdateSpecificKeyStatistics(DataType::kPKHashes, key.ToString(), statistic);
+  return s;
+}
+// TODO(DDD) field 带有过期时间。
+Status Redis::PKHMSetex(const Slice& key, const std::vector<FieldValueTTL>& fvts) {
+  uint32_t statistic = 0;
+  std::unordered_set<std::string> fields;
+  std::vector<FieldValueTTL> filtered_fvs;
+  for (auto iter = fvts.rbegin(); iter != fvts.rend(); ++iter) {
+    std::string field = iter->field;
+    if (fields.find(field) == fields.end()) {
+      fields.insert(field);
```

cppcheck

[performance] Searching before insertion is not necessary.

(stlFindInsert)